streaming_stt_nemo/__init__.py

```diff
@@ -12,6 +13,8 @@ class Model:
 
 
     def __init__(self, lang="en"):
+        # OVOS uses 'en-us' so this hacks that to work until dialects are supported
+        lang = lang[:2]
```

Comment:
There are a few 3-letter codes out there, could this be changed to `lang = lang.split('-')[0]`?